cmd/requester/README.md

```diff
@@ -1,77 +1,212 @@
+creationTimestamp: "2025-09-26T09:02:07Z"
+name: my-request-server
+namespace: default
+ownerReferences:
+  - apiVersion: v1
+    blockOwnerDeletion: true
+    controller: true
+    kind: Pod
+    name: my-request
+    uid: 15f63f7e-42ad-42b8-96a2-bf66f00cac26
+resourceVersion: "22498428"
+uid: 3e978564-d557-4f67-8f31-03f924d3686e
+$ kubectl get po my-request-server -oyaml | yq .spec.containers[0]
+command:
+  - vllm
+  - serve
+  - --port=8000
+  - /pvcs/local/default/vcp/hf/models--ibm-granite--granite-3.3-2b-instruct/snapshots/c4179de4bf66635b0cf11f410a73ebf95f85d506
+  - --max-model-len=32768
+env:
+  - name: CUDA_VISIBLE_DEVICES
+    value: "0"
+  - name: VLLM_CACHE_ROOT
+    value: /pvcs/shared/vcp/vllm
+image: docker.io/vllm/vllm-openai:v0.8.5
+imagePullPolicy: IfNotPresent
+name: inference-server
+ports:
+  - containerPort: 8080
+    protocol: TCP
+  - containerPort: 8081
+    protocol: TCP
+readinessProbe:
+  failureThreshold: 3
+  httpGet:
+    path: /health
+    port: 8000
+    scheme: HTTP
+  initialDelaySeconds: 60
+  periodSeconds: 5
+  successThreshold: 1
+  timeoutSeconds: 1
+resources:
+  limits:
+    cpu: "2"
+    memory: 6Gi
+    nvidia.com/gpu: "0"
+  requests:
+    cpu: "1"
+    memory: 250Mi
+    nvidia.com/gpu: "0"
+terminationMessagePath: /dev/termination-log
+terminationMessagePolicy: File
+volumeMounts:
+  - mountPath: /pvcs/local
+    name: local
+    readOnly: true
+  - mountPath: /pvcs/shared
+    name: shared
+  - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
+    name: kube-api-access-5kwgp
+    readOnly: true
+```
+
+Check the relayed readiness.
+```console
+$ kubectl wait pod/my-request-server --for=condition=Ready --timeout=120s
+pod/my-request-server condition met
+$ curl $REQ_IP:8080/ready
```

Comment:
This is pretty redundant with the `kubectl wait` just above.

Reply:
hmm, why is it redundant? The two commands says: Wait for the server running pod becomes ready, then the readiness is relayed to the server requesting pod.